jas/sast/sastscanner_test.go

```diff
@@ -62,15 +62,15 @@ func TestSastParseResults_ResultsContainIacViolations(t *testing.T) {
 	sastScanManager.resultsFileName = filepath.Join(jas.GetTestDataPath(), "sast-scan", "contains-sast-violations.sarif")
 
 	// Act
-	sastScanManager.sastScannerResults, err = jas.ReadJasScanRunsFromFile(sastScanManager.resultsFileName, jfrogAppsConfigForTest.Modules[0].SourceRoot, sastDocsUrlSuffix, scanner.MinSeverity)
+	vulnerabilitiesResults, _, err := jas.ReadJasScanRunsFromFile(sastScanManager.resultsFileName, jfrogAppsConfigForTest.Modules[0].SourceRoot, sastDocsUrlSuffix, scanner.MinSeverity)
```

Comment:
if this test case is suppose to check violations, shouldn't the results be in the second returned value?
If we are checking here about vulnerabilities please change the name of the function.
should we add a test case for vulnerabilities and a test case for violations?